infrastructure/logging/src/main/resources/log4j2.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration level="INFO">
     <Loggers>
+        <!--         Remove all DEBUG and TRACE logging from noisy dependencies-->
+        <Logger name="io.netty" level="INFO" additivity="FALSE"/>
```

Comment:
It affects default logging behavior (eliminating 3rd libraries debug+ logging from our logs). @rolfyone do you confirm this change?